charts/core-crd/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: core-crd
 description: A Helm chart for Kubernetes
 type: application
-version: 2.8.5
-appVersion: "v2.8.5"
+version: 2.8.4
```

Comment:
@jasmingacic looks like you bumped the versions here directly instead of in the other chart - also you've just released operator 2.8.5 so this chart will not be re-released with it...

Reply:
check the next commit please